RELEASE/scripts/autoscend/iotms/mr2024.ash

```diff
@@ -167,3 +167,17 @@ skill dartSkill()
 	}
 	return to_skill(7513); // If there aren't any darts available return the Darts: Throw at %PART1
 }
+
+boolean dartEleDmg()
+{
+	string[int] perks;
+	perks = split_string(get_property("everfullDartPerks").to_string().to_lower_case(), ",");
+	foreach perk in perks
```

Comment:
can the foreach be removed and instead `contains_text(perks, "add ")`

Reply:
Yes, this can be greatly simplified. Modeled it off of my Everything Looks Red CD reduction but that was way fancier than it needed to be.